src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/IsOppfolgingstilfelleClient.kt

```diff
@@ -0,0 +1,82 @@
+package no.nav.syfo.oppfolgingstilfelle
+
+import no.nav.syfo.NAV_CALL_ID_HEADER
+import no.nav.syfo.auth.bearerHeader
+import no.nav.syfo.auth.tokendings.TokendingsClient
+import no.nav.syfo.createCallId
+import no.nav.syfo.logger
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestClientResponseException
+import org.springframework.web.client.RestTemplate
+import java.time.LocalDate
+
+@Service
+class IsOppfolgingstilfelleClient(
+    private val tokenDingsClient: TokendingsClient,
+    @Value("\${isoppfolgingstilfelle.url}") private val baseUrl: String,
+    @Value("\${isoppfolgingstilfelle.id}") private var targetApp: String,
+) {
+    private val log = logger()
+
+    fun isSykmeldt(token: String): Boolean {
```

Comment:
Ikke akkurat mye logikk her, men om du vil kan du jo kaste på en enhetstest eller to for å sjekke at sykmelding "logikken" er som forventet

Reply:
Jeg har lagt til en test på denne. Jeg måtte fjerne ApplicationTest midlertidig, fordi den ikke fungerer. Testene var disabled fra før, så den har uansett ikke kjørt. Jeg har prøvd å fikse den en stund nå uten å få det til å spille helt. Tenker at jeg tar det i egen PR.